features/addons/src/main/feature/feature.xml

```diff
@@ -3,10 +3,13 @@
 
 	<!-- these are 2.x add-ons in separate repositories (which do not have their own Karaf feature defined), so we include them here -->
 
-    <feature name="openhab-binding-zigbee" description="ZigBee Binding" version="${project.version}">
+    <feature name="openhab-binding-zigbee-ember" description="ZigBee Binding for Ember NCP" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-serial</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.zigbee/${project.version}</bundle>
+        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.zigbee.ember/${project.version}</bundle>
+        <bundle start-level="80">mvn:com.zsmartsystems.zigbee/com.zsmartsystems.zigbee</bundle>
```

Comment:
Hm, that won't really work - addons must not bring any external dependencies that are not part of the openhab base feature (or available in the TP of the IDE). It'll be better to continue to include them as a lib in the appropriate version, I guess.

Reply:
I thought as much, but hoped there might have been another setting somewhere to set the repo. I can look at adding the files to the JARs, or something similar, but I see you have other comments on the overall refactoring that we probably need to discuss further first ;)